### ISSUE TITLE
Removing the composer entry will give an error the next time composer install is done

### ISSUE THREAD

**Issue by koden-km** [eloquent/composer-npm-bridge#5]
I added the composer npm bridge to my project in a feature branch. Did `composer update` everything was fine. After switching back to develop branch and running `composer install` to install packages for the develop version, i got an error.

```
  - Removing eloquent/composer-npm-bridge (2.1.0)

Fatal error: Class 'Eloquent\Composer\NpmBridge\NpmBridge' not found in /Users/xxxx/Dev/project/vendor/eloquent/composer-npm-bridge/src/NpmBridgeFactory.php on line 33
```

**Comment by garex** [comment]
It's this issue: https://github.com/composer/composer/issues/4520

Workaround is to preload classes. I will try it now.

**Comment by ezzatron** [comment]
Fixed in [4.0.1](https://github.com/eloquent/composer-npm-bridge/releases/4.0.1).
